src/java.security.jgss/share/native/libj2gss/gssapi.h

```diff
@@ -43,7 +43,7 @@
 extern "C" {
 #endif /* __cplusplus */
 
-#if defined(TARGET_OS_MAC)
+#if TARGET_OS_MAC && (defined(__ppc__) || defined(__ppc64__) || defined(__i386__) || defined(__x86_64__))
```

Comment:
We can simplify that check for `#if TARGET_OS_MAC && !defined(__aarch64__)`

Reply:
Not sure
it's better to be in line with what is present in xcode, to make it easy to track changes if they happen in future.

Reply:
Could you please add a quick comment with a reference to what you just linked? It doesn't look clear out of the box but it makes a lot of sense.